modules/aws/Makefile

```diff
@@ -1,7 +1,18 @@
 export AWSCLI_VERSION ?= 1.11.185
 
+WITH_AWS ?= aws-vault exec $(AWS_PROFILE) --
+
 ## Install aws cli bundle
 aws/install:
 	pip install --no-cache-dir --disable-pip-version-check awscli==$(AWSCLI_VERSION)
```

Comment:
:(

but i guess it's necessary? thing is `pip` command is not-standard. sometimes it's `pip3`

Reply:
As you see I did not add target `aws/install`. 
Probably that should be deprecated because we have `packages/%`?

Reply:
Currently, `cloudposse/packages` only handles single-file executables